src/components/TradeWidget/OrderValidity.tsx

```diff
@@ -184,14 +184,18 @@ const OrderValidity: React.FC<Props> = ({
       const sanitizedUntilValue = makeMultipleOf(5, validUntilInputValue).toString()
 
       batchedUpdates(() => {
+        if (sanitizedFromValue === '0') setAsap(true)
```

Comment:
I find strange that we need two vars for the from and the until. I mean, For example the `asap`, couldn't just be derived from the value of `validFrom` instead of having a hook for it's state?

Reply:
this is true, i'll look into it for after the merge

Reply:
but it is also used for the `checkbox`es next to the inputs, you can set the `validFrom` by checking the box so it needs a toggle

Reply:
or you mean setting the form to zero on checkbox change? :thinking: that also makes sense